Gemfile.lock

```diff
@@ -66,15 +66,14 @@ GEM
     bindex (0.8.1)
     bootsnap (1.9.1)
       msgpack (~> 1.0)
-    brakeman (5.1.2)
+    brakeman (5.1.1)
```

Comment:
[concern] I think that something went awry with a merge, because it looks like a lot of changes from the main branch are getting walked back in this commit!